include/psa/crypto_values.h

```diff
@@ -2726,4 +2733,17 @@ static inline int mbedtls_svc_key_id_is_null( mbedtls_svc_key_id_t key )
 
 /**@}*/
 
+/**@}*/
+
+/** \defgroup interruptible Interruptible operations
+ * @{
+ */
+
+/** Maximum value for use with \c psa_interruptible_set_max_ops() to determine
+ *  the maximum number of basic operations allowed to be executed by an
+ *  interruptible function in a single call. */
+#define PSA_INTERRUPTIBLE_MAX_OPS_MAX INT32_MAX
```

Comment:
“max ops max” is an awkward name. Semantically, this isn't a maximum: it means infinite. `PSA_INTERRUPTIBLE_ALWAYS_COMPLETE`? `PSA_INTERRUPTIBLE_MAX_OPS_UNBOUNDED?`